request.js

```diff
@@ -1055,6 +1056,14 @@ Request.prototype.start = function () {
       }
     }
 
+    if (self.keyLog) {
+      socket.on('keylog', (line) => {
```

Comment:
Should we only attach this when the socket is fresh (`!resusedSocket`)? Since we are not disposing this listener we'll keep on attaching a new handler every the socket is reused (given that execution has keyLog option set)